### ISSUE TITLE
Tag picker issue : When "Remove" button receives the keyboard focus, the color contrast ratio of the button with respect to background is less than 3:1.

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#17625]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^7.123.4",
- **Browser and OS versions**: Chrome and EDGE , Windows 10

### Describe the issue:
When "Remove" button receives the keyboard focus in Tag picker control , the color contrast ratio of the button with respect to background is less than 3:1.

#### Actual behavior:
When "Remove" button receives the keyboard focus in Tag picker control , the color contrast ratio of the button with respect to background is less than 3:1.

#### Expected behavior:
When "Remove" button receives the keyboard focus in Tag picker control , the color contrast ratio of the button with respect to background should be more than or equal to
<img width="960" alt="Tagpicker" src="https://user-images.githubusercontent.com/48669559/113001865-d9c6cc80-918e-11eb-8c25-c88d764a8afd.png">
 3:1.

**Comment by naksenzh** [comment]
Hi Team, any ETa for this bug please? @smhigley
